front/app/components/UI/QuillEditor/index.tsx

```diff
@@ -438,7 +446,11 @@ const QuillEditor = memo<Props & InjectedIntlProps>(
           theme: 'snow',
           placeholder: placeholder || '',
           modules: {
-            blotFormatter: !noImages || !noVideos ? true : false,
+            blotFormatter: !noVideos
```

Comment:
The blot formatter is used for video now, image alignment is handled by the new module.